app/cells/plugins/core/cell.rb

```diff
@@ -2,6 +2,10 @@ module Plugins
   module Core
     class Cell < FieldCell
       view_paths << "#{Cortex::Plugins::Core::Engine.root}/app/cells"
+
+      def required?
+        field_item.field.validations["presence"] == true
```

Comment:
Because presence is what Rails / ActiveModel use to reference required validations - basically it's just to keep it consistent with how Rails does things and not change too many keywords